### PR TITLE
Use createMany in Seed database example

### PR DESCRIPTION
**Pull request by simoncrypta**
This replaces the example on api/db/seed.js with createMany, a preview feature in Prisma. 

Like @dthyresson said on issue #1771 
> By using createMany one no longer has to iterate over a set of records to load data -- and no longer has to worry about Promises resolving per record creation.

I think is already a good idea to show this example in the comment section.

### CODE REVIEW

File: packages/create-redwood-app/template/api/db/seed.js [hunk 1]
```diff
@@ -12,11 +12,31 @@ async function main() {
...
+        { email: "mark@example.com" },
+        { email: "jackie@example.com" },
+        { email: "bob@example.com" },
+      ],
+      skipDuplicates: true, // Supported with Postgres database
+    })
+    console.log(`Created ${result.count} users!`)
+
+  */
+  // Note: This example use createMany in preview.
```
Comment (dthyresson) [inline packages/create-redwood-app/template/api/db/seed.js:29]: Let's change the comment to:

```
Note: createMany creates multiple records in a transaction. 

To enable this feature, add createMany to previewFeatures in your schema. 

See: https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#createmany-preview

Note: createMany is not supported by SQLite.
```

File: packages/create-redwood-app/template/api/db/seed.js [hunk 2]
```diff
@@ -12,11 +12,31 @@ async function main() {
   // Ideally this file should be idempotent: running it multiple times
   // will result in the same database state (usually by checking for the
   // existence of a record before trying to create it). For example:
-  //
```
Comment (dthyresson) [inline packages/create-redwood-app/template/api/db/seed.js:15]: I just realized that 

createMany is not supported by SQLite

therefore we will still need the singular:

`await db.user.create`

examples back with some comment that ways to use with SQLite vs Postgres as options.

**Review by dthyresson — CHANGES_REQUESTED** [review]

File: packages/create-redwood-app/template/api/db/seed.js (hunk 2 above)
Comment (simoncrypta) [reply packages/create-redwood-app/template/api/db/seed.js:15]: Oh yeah, same; I just noticed the Remarks section. 
To clarify, you think that we should show 2 different examples?

**Comment by thedavidprice** [comment]
@dthyresson I think this one's awaiting your guidance.

**Review by dthyresson — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**